debian/ubuntu-advantage-tools.postinst

```diff
@@ -114,15 +114,15 @@ redact_ubuntu_release_from_ua_apt_filenames() {
 check_service_is_enabled() {
     service_name=$1
     _RET=$(/usr/bin/python3 -c "
-import os
-import json
-from uaclient.config import UAConfig
-cfg = UAConfig()
-status = cfg.read_cache('status-cache')
-if status:
-    for service in status.get('services', []):
-       if service.get('name', '') == '${service_name}':
-           print(service.get('status', ''))
+from uaclient.files.state_files import status_cache_file
+try:
+    status = status_cache_file.read()
+    if status:
+        for service in status.get('services', []):
+            if service.get('name', '') == '${service_name}':
+                print(service.get('status', ''))
+except Exception:
+    print('Error checking service: ${service_name}')
```

Comment:
I guess this won't actually print to stdout, the value will just be used in the following if statement. Maybe it should instead be
```suggestion
    # Assume not enabled
    pass
```